### PR TITLE
fix bugs for timer saver

### PR DESCRIPTION
**Pull request by wary**
### What is this PR for?

* add saveNote action when cron schedule setting in  file notebook.controller.js
   * case: 
  open one note，edit some text in a paragraph, then do a cron schedule setting, the content which   just edited disappeared

* change invoke method for the timer saver  in file paragraph.controller.js
  * case:
   open one browser，edit some text in a paragraph, then open the note in  other browser，you will find that the text you just edited did not be synced

### What type of PR is it?
[Bug Fix]

### Todos
* [ ] - Task

### What is the Jira issue?
[ZEPPELIN-2323](https://issues.apache.org/jira/browse/ZEPPELIN-2323)

### How should this be tested?
Outline the steps to test the PR here.

### Questions:
* Does the licenses files need update?
* Is there breaking changes for older versions?
* Does this needs documentation?

### CODE REVIEW

File: zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js [hunk 1]
```diff
@@ -599,7 +599,9 @@ function ParagraphCtrl($scope, $rootScope, $route, $window, $routeParams, $locat
     var session = editor.getSession();
     var dirtyText = session.getValue();
     $scope.dirtyText = dirtyText;
-    $scope.$broadcast('startSaveTimer');
+    //the saveTimer is not be invoked with this statement correctly
```
Comment (1ambda) [inline zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js]: Could you remove comment in line 602, 603?

**Comment by 1ambda** [comment]
Hi @wary  Thanks for contribution.

- Zeppelin build system relies on travis. So please setup your travis account and travis project for Zeppelin. 
- https://zeppelin.apache.org/contribution/contributions.html#continuous-integration
- The repository name must be `zeppelin` (not `incubator-zeppelin`, ...)
- It's not mandatory to pass all tests to be reviewed. But some reviewers might select passed PRs first

Please trigger travis CI by pushing new commits or amend your last commit (`git commit --amend` and  `git push your-remote HEAD --force`) after setup your travis.

**Comment by soralee** [comment]
Thanks @wary for fixing this!
But, could you explain detail this PR such as `what is this PR for` and `How should this be test`?

**Comment by wary** [comment]
ok, After get off work，I will remove the comments and explain detail for this PR, thx

**Comment by soralee** [comment]
@wary No worry! It's okay! And thanks and Congrats for your contribution. :smile: 
Let me test it out and give you feedback.

File: zeppelin-web/src/app/notebook/notebook.controller.js [hunk 2]
```diff
@@ -398,6 +398,7 @@ function NotebookCtrl($scope, $route, $routeParams, $location, $rootScope,
     } else {
       $scope.note.config.cronExecutingUser = '';
     }
+    $scope.saveNote();
```
Comment (1ambda) [inline zeppelin-web/src/app/notebook/notebook.controller.js]: Is this change is related with **fix bugs for timer saver**?

Comment (wary) [inline zeppelin-web/src/app/notebook/notebook.controller.js]: I think it's a enhancement for timer saver, as we know, if timer saver works well, the paragraph will be saved in 10 seconds， There is a chance that the edited content will lost if it isn't saved in time after GET_NOTE Operation caused. the cron schedule setting cause the GET_NOTE operation, so I think note should be saved in this place.

Comment (1ambda) [inline zeppelin-web/src/app/notebook/notebook.controller.js]: it makes sense.

**Comment by Leemoonsoo** [comment]
@wary Thanks for the contribution!
LGTM and merge to master and branch-0.7 if no further comments.
